src/Application/PHPStanRunner.php

```diff
@@ -71,7 +71,7 @@ public function run(
         $process = new Process(array_filter([
             PHP_BINARY,
             $xdebugMode ? '-dzend_extension=xdebug.so' : null,
-            'vendor/bin/phpstan',
+            getcwd() . '/vendor/bin/phpstan',
```

Comment:
I wonder why this was done. Because we already pass the working directory on line 86.

Reply:
I don't know why, with my GenerationBaselineTest.php I need this, otherwise something goes wrong (Could not open input file: vendor/bin/phpstan)